src/materials/ConcreteThermalMoisture.C

```diff
@@ -71,25 +71,34 @@ ConcreteThermalMoisture::validParams()
   params.addParam<MooseEnum>(
       "aggregate_pore_type", aggregate_pore_type, "aggregate pore structure");
 
-  params.addParam<Real>("cement_mass", "cement mass (kg) per m^3");
-  params.addParam<Real>("aggregate_mass", "aggregate mass (kg) per m^3");
+  // WGA - unclear if units for following two params is kg or kg/m^3 - assuming kg
+  params.addParam<Real>("cement_mass", "cement mass per m^3");
+  params.setDocUnit("cement_mass", "kg");
+  params.addParam<Real>("aggregate_mass", "aggregate mass per m^3");
+  params.setDocUnit("aggregate_mass", "kg");
   params.addParam<Real>("water_to_cement_ratio", "water to cement ratio");
   params.addParam<Real>("aggregate_vol_fraction", "volumetric fraction of aggregates");
   params.addParam<Real>("concrete_cure_time", "concrete curing time in days");
-  params.addParam<Real>("ref_density", "refernece density of porous media Kg/m^3");
-  params.addParam<Real>("ref_specific_heat", "reference specific heat of concrete J/Kg/0C");
-  params.addParam<Real>("ref_thermal_conductivity",
-                        "concrete reference thermal conductivity (W/m/C)");
+  params.addParam<Real>("ref_density", "refernece density of porous media");
+  params.setDocUnit("ref_density", "kg/m^3");
+  params.addParam<Real>("ref_specific_heat", "reference specific heat of concrete");
+  params.setDocUnit("ref_specific_heat", "J/kg/0C");
```

Comment:
```suggestion
  params.setDocUnit("ref_specific_heat", "J/kg/K");
```
Note that the size of a temperature increment in K is the same as that in C, and I'd prefer to using K as much as possible here. For that same reason I'm suggesting changing the thermal conductivity below as well.